public/locales/nn/translation.json

```diff
@@ -13,7 +13,7 @@
     "pensjonsbeholdning-forklart-tekst": "Du opparbeider pensjonsbehaldning frå og med det første året du har pensjonsgivande inntekt. Behaldninga aukar for kvart år basert på behaldninga det føregåande året, pensjonsgivande inntekt to år tilbake i tid og eventuell anna opptening av pensjonsrettar, og også ei årleg regulering. Du kan bruke pensjonsbehaldninga som utgangspunkt for å berekne alderspensjonen du får frå NAV når du pensjonerer deg.",
 
     "pensjonspoeng-forklart": "Pensjonspoeng forklart",
-    "pensjonspoeng-forklart-tekst": "Pensjonspoeng blir rekna av pensjonsgivande inntekt, først og fremst arbeidsinntekt og personinntekt frå næringsverksemd kvart år, frå og med det året du fyller 17, til og med det året du fyller 69. Pensjonspoeng blir brukt som utgangspunkt for å berekne alderspensjonen du får frå NAV når du pensjonerer deg.",
+    "pensjonspoeng-forklart-tekst": "Pensjonspoeng blir rekna av pensjonsgivande inntekt, først og fremst arbeidsinntekt og personinntekt frå næringsverksemd. For personer født til og med 1942 opptjenes pensjonspoeng frå og med det året ein fyller 17 til og med det året ein fyller 69. For personer født frå 1943 til og med 1962 opptjenes pensjonspoeng frå det året ein fyller 17 til og med det året ein fyller 75.",
```

Comment:
"For personer født" -> "For personar fødde"

Når det gjelder "opptjenes pensjonspoeng" tror jeg ikke man kan bruke slik passivform på nynorsk, men må bruke f.eks. "blir pensjonspoeng opptent".

Reply:
Tusen takk, endrer